src/pages/workspace/WorkspacePeoplePage.js

```diff
@@ -0,0 +1,328 @@
+        );
+    }
+
+    render() {
+        return (
+            <ScreenWrapper>
+                <HeaderWithCloseButton
+                    title={this.props.translate('common.people')}
+                    onCloseButtonPress={() => Navigation.dismissModal(true)}
+                />
+                <ConfirmModal
+                    danger
+                    title={this.props.translate('workspace.people.removeMembersTitle')}
+                    isVisible={this.state.isRemoveMembersConfirmModalVisible}
+                    onConfirm={this.removeUsers}
+                    onCancel={this.hideConfirmModal}
+                    prompt={this.props.translate('workspace.people.removeMembersPrompt')}
+                    confirmText={this.props.translate('common.remove')}
+                    cancelText={this.props.translate('common.cancel')}
+                />
+                <ScrollView style={[styles.settingsPageBackground]} bounces={false}>
+                    <View style={styles.pageWrapper}>
+                        <Text style={[styles.mb6, styles.textP, styles.textFull]}>
+                            {this.props.translate('workspace.people.tagline')}
+                        </Text>
+                        <View style={styles.buttonRow}>
+                            <Button
+                                success
+                                style={[]}
+                                text={this.props.translate('common.invite')}
+                                onPress={this.inviteUser}
+                            />
+                            <Button
+                                danger
+                                style={[styles.ml2]}
+                                isDisabled={this.state.selectedEmployees.length === 0}
+                                text={this.props.translate('common.remove')}
+                                onPress={this.askForConfirmationToRemove}
+                            />
+                        </View>
+                        <View style={[styles.w100, styles.mt4]}>
+                            {
+                                this.props.policy.employeeList ? (
+                                    <FlatList
+                                        ListHeaderComponent={this.renderHeader()}
+                                        ref={flatListRef}
+                                        renderItem={this.renderItem}
+                                        contentContainerStyle={[styles.w100]}
+                                        initialRowHeight={32}
+                                        // eslint-disable-next-line max-len
+                                        data={(this.props.policy.employeeList && this.props.policy.employeeList.length !== 0)
+                                            // eslint-disable-next-line max-len
+                                            ? this.props.policy.employeeList.map(email => this.props.personalDetails[email] || false)
```

Comment:
I think we will have an array with values like this ?

```js
['someone@exists.com', false, 'another@inpersonaldetails.com'];
```

What can we do for the people who were not found? Should we just filter them out? Show some default?

Reply:
I have filtered them out for now. However, if users signs up using phone number, they might not have `email`, right? 

Should we show their mobile number instead?

Reply:
Ah, the answer is no. `email` is kind of a misleading/unfortunate variable name in this case. 

Even exclusive phone users have an `email` and it looks something like:

`+15141231234@expensify.sms`

Reply:
Thank you! Great to know.

Do you reckon we should filter out these sm`expensify.sms` emails?

Reply:
I can't of a reason why we would. If it's possible to invite someone with a phone number to a workspace then we would show them in this list..

Reply:
Should we then consider changing the header name of the column to be `Email or phone` same as the Invite modal has in placeholder of the input?

Reply:
It might be a good follow up issue, but I don't think we need to block on it.